src/_guides/language/effective-dart/documentation.md

```diff
@@ -267,11 +267,58 @@ int weekday;
 int get checkedCount => ...
 {% endprettify %}
 
+### PREFER starting boolean variables or properties with "Whether" followed by noun or gerund phrases.
+
+The doc comment should clarify the states this variable represents. 
+This is true even for getters which may do calculation or other work. 
+What the caller cares about is the *result* of that work, not the work itself.
+
+{:.good}
+<?code-excerpt "docs_good.dart (noun-phrases-for-non-boolean-var-etc)"?>
```

Comment:
Seems this is the boolean excerpt, which is probably why the excerpt check is failing now:

```suggestion
<?code-excerpt "docs_good.dart (noun-phrases-for-boolean-var-etc)"?>
```